src/commcare_cloud/ansible/roles/formplayer/templates/application.properties.j2

```diff
@@ -7,7 +7,7 @@ commcarehq.environment={{ env_monitoring_id }}
 commcarehq.formplayerAuthKey={{ FORMPLAYER_INTERNAL_AUTH_KEY }}
 server.port={{ formplayer_port }}
 user.suffix=commcarehq.org
-logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=INFO
+logging.config={{ _formplayer_target_dir }}/logback-spring.xml
```

Comment:
I think you want `{{ formplayer_release_dir }}` here instead. `_formplayer_target_dir` [sometimes is a path that includes a symlink to the current release and sometimes is an absolute path](https://github.com/dimagi/commcare-cloud/blob/70c9666692762dc8ccc23e1e91918f936d97607d/src/commcare_cloud/ansible/roles/formplayer/vars/main.yml#L13-L15) (with no symlinks). Here we always want the absolute path.

Reply:
Hmm, why wouldn't we want the symlinked current directory? If the `formplayer` process is instantiated with the current release [scope](https://github.com/dimagi/commcare-cloud/blob/261e56ccb8ba55ae89cf8d0e68ae589639ebffbb/src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_formplayer_spring.conf.j2#L4) shouldn't all references be scoped to the current release?

Reply:
I guess there's two things here. `_formplayer_target_dir` will sometimes be the one with the symlink (`formplayer_current_dir`=`{{ formplayer_build_dir }}/current`) and sometimes the real path (`formplayer_release_dir`=`{{ formplayer_build_dir }}/releases/{{ new_release_name }}`), so we should replace it with one or the other of those so that it's consistent between runs. (Why does it differ between runs? If you're doing a new deploy (`cchq <env> deploy formplayer`) then we first create a new release and this gets run in that context, so `_formplayer_target_dir` will be the new release; if you're running `cchq <env> update-config` then we want to affect the current directory in place, so it'll be the `current/` directory.)

Now, on this line, the question is do we want all releases to point to the `current/` directory's logging config, or to their own logging config? To your point, since formplayer is only ever run from the current directory, in practice it's a moot point. However since we do have to decide between the two, I'd say a release should point to its own logging config rather than current's logging config—it just makes more sense. I.e. if it ever _were_ to matter (like if formplayer were run directly from a release) running its own logging config would be the correct behavior.

Reply:
Ahh that makes sense, updated.